common/src/commonMain/kotlin/com/cuhacking/atlas/db/SharedDatabase.kt

```diff
@@ -1,38 +1,30 @@
 package com.cuhacking.atlas.db
 
+import com.cuhacking.atlas.common.CoroutineDispatchers
+import com.cuhacking.atlas.common.DataCache
+import com.cuhacking.atlas.common.FeatureApi
 import com.squareup.sqldelight.db.SqlDriver
+import io.ktor.client.HttpClient
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
 
 lateinit var database: Database
 
 expect fun provideDbDriver(): SqlDriver
 
 fun SqlDriver.createDatabase(): Database {
-    // Uncomment the following code and delete the rest. It was used for testing only
-//    return Database(
-//            driver = this,
-//            featureAdapter = Feature.Adapter(
-//                    jsonAdapter = GeoJsonAdapter
-//            )
-//    )
-//    Database.Schema.create(this)
-    database = Database(
-        driver = this,
+    return Database(
+            driver = this,
             featureAdapter = Feature.Adapter(
                     jsonAdapter = GeoJsonAdapter
             )
     )
-    database.featureQueries.deleteAll()
-    database.featureQueries.clearFts()
-
-    val sampleFeature = GeoJsonAdapter.decode("""{"type":"Feature","geometry":{"type":"Point",
-        |"coordinates":[-75.0,45.0]},"properties":{}}""".trimMargin())
-    val feature1 = Feature(1, "River Building", "Richcraft Hall", "Building", null, null, null, sampleFeature)
-    val feature2 = Feature(2, "RB 2311", null, "room", "River Building", "RB2", null, sampleFeature)
-    @Suppress("MagicNumber")
-    val feature3 = Feature(3, "HS 1301B", null, "room", "Health Sciences Building", "HS1", null, sampleFeature)
-    database.featureQueries.insertFeature(feature1)
-    database.featureQueries.insertFeature(feature2)
-    database.featureQueries.insertFeature(feature3)
+}
 
-    return database
+ fun populateDatabase() {
+     val httpClient = HttpClient()
+     val dataCache = DataCache(CoroutineDispatchers)
```

Comment:
We'll want to create singleton instances of these classes (i.e. just a global variable, for now) since we want to avoid creating multiple instances of these classes.
Notably with the `DataCache`, since it'll store the data in-memory on JS, if we create a new `DataCache` later on (like to retrieve the data) then we'll lose that in-memory data.